### PR TITLE
Add a test page with a peer connection from realistic video.

### PR DESCRIPTION
**Pull request by ehlemur**
**Description**
Add a test page with a stream from a realistic video to a peer connection.

**Purpose**
Currently, telemetry tests for WebRTC use an auto generated video that is not realistic and is very simple to encode and decode. This might be a way for using realistic video for telemetry tests.

### CODE REVIEW

File: src/manual/peer2peer-video/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2017 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<html><head>
+  <title>Video to peer connection</title>
```
Comment (KaptenJansson) [inline src/manual/peer2peer-video/index.html]: peer connection > peerConnection

File: src/manual/peer2peer-video/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <link rel="stylesheet" href="../css/main.css">
+</head>
+<body>
+  <div id="container">
+    <h1>Stream from video to peer connection</h1>
```
Comment (KaptenJansson) [inline src/manual/peer2peer-video/index.html]: peer connection > peerConnection

**Comment by ehlemur** [comment]
PTAL

**Comment by ehlemur** [comment]
Thanks :)
Do you have to publish it, or something?

**Comment by KaptenJansson** [comment]
Yes it's currently served on AppEngine (want to move these to a separate repo soonish.)

**Pull request merged**